### ISSUE TITLE
Download page: Add an option to downlaod releases

### ISSUE THREAD

**Issue by RunDevelopment** [PrismJS/prism#1870]
We should add an option to download specific releases of Prism instead of the latest version from the `master` branch for the following reasons:

1. The current version in `master` might not be the most stable version (or even pass the build).
2. The Autoloader SRI plugin (#1851). The hashes of the plugin are specific for the current version of the build. This means that when you download the latest version of Prism with Autoloader + SRI and use your favorite CND with Prism v1.XX, then the hashes of all files which have been changed since the last release will be incorrect causing Autoloader to be unable to load quite a few languages.
   This addition to the download page is basically a necessity before we release Autoloader SRI.

**Comment by mAAdhaTTah** [comment]
For 1, it definitely _should_ pass the build. Given that we've had the download page pointing to `master` since the beginning, it's always been our intention that `master` should be stable & downloadable.

2 makes things more interesting though. Would have to think about that.

**Comment by RunDevelopment** [comment]
> For 1, it definitely should pass the build.

True, and even if it fails, it will most likely still be downloadable.
